.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish to crates.io
+
+on:
+  push:
+    branches:
+      main
+
+jobs:
+  publish:
+    # need to rename environment name after this is merged
+    environment: Autobump version
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Cardinal-Cryptography/AlephBFT'}}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: force publish if version bumped
+        run: |
+          if [ -n "$(git diff HEAD~ -- Cargo.toml | grep '^+version =')" ]; then
+            echo "Version in Cargo.toml was bumped in this PR, publishing to crates.io."
+            touch publishMe
```

Comment:
I believe this trick is not necessary anymore, just do `credentials` always, and move the above `if` to `publish`.

It was needed, because of some silly dependencies between this and the forced version bump.

Reply:
Right, I have not noticed that this can be simplified. Will change.